frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -497,6 +497,9 @@ $monitoring-line-height: 18px;
 .query-browser__zoom {
   cursor: ew-resize;
   position: relative;
+  &:focus {
+    outline: none;
```

Comment:
I'm not sure we want to remove the focus indicator. That's generally bad for accessibility. How does it look without this style?

Reply:
I do think removing the focus outline looks better and I'm not sure that the focus styling is actually beneficial for accessibility in this case?

With focus outline:
![screenshot](https://user-images.githubusercontent.com/460802/95946677-03ca4380-0e28-11eb-9ffd-e7cf388863ee.png)

Without focus outline:
![screenshot](https://user-images.githubusercontent.com/460802/95947359-6112c480-0e29-11eb-8299-089da45405e2.png)

Reply:
The problem is, if you're tabbing through elements on the page, the focus ring will disappear when you get once you get to the graph, and you won't know where you are on the page.

Reply:
I think we're OK here. Setting `tabIndex` to `-1` prevents selecting the element with the keyboard.

From https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex:
> A negative value (usually tabindex="-1") means that the element is not reachable via sequential keyboard navigation, but could be focused with Javascript or visually by clicking with the mouse.